### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by flyingrub**
it should have the installation files ?

### CODE REVIEW

File: contrib/french.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		<license><![CDATA[http://opensource.org/licenses/gpl-license.php GNU General Public License v2]]></license>
+		<title lang="en"><![CDATA[Reputation System]]></title>
+		<description lang="en"><![CDATA[French language pack for Reputation System]]></description>
+		<author-group>
+			<author>
+				<username><![CDATA[FlyinGrub]]></username>
+				<homepage><![CDATA[https://github.com/flyingrub/ ]]></homepage>
+			</author>
+		</author-group>
+		<mod-version>0.6.3</mod-version>
```
Comment (Pico) [inline contrib/french.xml]: It should be 0.7.0

**Comment by Pico** [comment]
Also add to install_mod.xml `<link type="language" href="./contrib/french.xml" lang="en">French Language Pack</link>`
below
`<link type="language" href="./contrib/polish.xml" lang="en">Polish Language Pack</link>`

;)

**Comment by flyingrub** [comment]
I will :)

**Comment by flyingrub** [comment]
my webmaster didn't want this mod anymore ..

**Pull request closed without merging**